src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetUptime.cs

```diff
@@ -0,0 +1,82 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Management.Automation;
+using System.Diagnostics;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// This class implements Get-Uptime.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "Uptime", DefaultParameterSetName = "Timespan", HelpUri = "")]
+    public class GetUptimeCommand : PSCmdlet
+    {
+        /// <summary>
+        /// Timespan parameter
+        /// Time since the system started up
+        /// </summary>
+        /// <value></value>
+        [Parameter(ParameterSetName = "Timespan")]
+        public SwitchParameter Timespan { get; set; } = new SwitchParameter();
+
+        /// <summary>
+        /// Since parameter
+        /// The system startup time
+        /// </summary>
+        /// <value></value>
+        [Parameter(ParameterSetName = "Since")]
+        public SwitchParameter Since { get; set; } = new SwitchParameter();
+
+        /// <summary>
+        /// Pretty parameter
+        /// Convert output to default formated string
+        /// </summary>
+        /// <value></value>
+        [Parameter(ParameterSetName = "Timespan")]
+        [Parameter(ParameterSetName = "Since")]
+        public SwitchParameter Pretty { get; set; } = new SwitchParameter();
```

Comment:
Do we need this parameter?

With strongly typed output, we can always use pretty formatting and still have useful output.